fxdc/writedata/parseobject.py

```
@@ -39,38 +42,60 @@ def convertobject(
                         convertedobject = data
 
         debug("Converted Object:", convertedobject, "Type:", type_)
-        return type_, convertedobject
+        return type_, convertedobject, descriptions
 
-    def parse(self, tab_count: int = 0, dataobject: object = None) -> str:
+    def parse(
+        self,
+        tab_count: int = 0,
+        dataobject: object = None,
+        descs: dict[str, str] = None,
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)